include/openthread/platform/ble.h

```diff
@@ -281,6 +315,21 @@ otError otPlatBleGattServerIndicate(otInstance *aInstance, uint16_t aHandle, con
  */
 extern void otPlatBleGattServerOnWriteRequest(otInstance *aInstance, uint16_t aHandle, const otBleRadioPacket *aPacket);
 
+/**
+ * Function to retrieve from platform BLE link capabilities.
+ *
+ * @param[in] aInstance             The OpenThread instance structure.
+ *
+ */
+otBleLinkCapabilities otPlatGetBleLinkCapabilities(otInstance *aInstance);
+
+/**
+ * Function to retrieve from platform multiradio support of BLE and IEEE.
+ *
+ * @param[in] aInstance             The OpenThread instance structure.
+ *
+ */
+bool otPlatMultiradioSupport(otInstance *aInstance);
```

Comment:
Same for this API. Use `otPlatBe` prefix.

Suggest `otPlatBleSupportsMultiRadio()`.

Reply:
fixed.